### PR TITLE
fix sol

### LINKED ISSUE #2560: Code issue for CF - Power Products solution
**Module**: Gold-Divisibility
**Topic**: Small issues in the suggested code
**Message**: I was practicing this topic and according to some bugs in my code , I went to check the suggested code for this problem and i saw 2 little issues in code that causes wrong answer for that solution.

- `int size` for variable n of the main codeforces problem , because of .size() method for vectors , string and ... ,  makes compile error
- in the line 43 `vector<pi> inverse(size);` should be change to `vector<pi> inverse(number_of_primes);` to get the code works correct.

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

should close #2560 
too lazy to sansify

### CODE REVIEW

File: src/components/TopNavigationBar/TopNavigationBar.tsx [hunk 1]
```diff
@@ -182,9 +181,12 @@ export default function TopNavigationBar({
...
             <div className="pr-16 sm:text-center sm:px-16">
               <p className="font-medium text-white">
-                <span className="md:hidden">Register for the USACO January Workshop!</span>
+                <span className="md:hidden">
+                  Register for the USACO January Workshop!
+                </span>
                 <span className="hidden md:inline">
-                  USACO Bronze/Silver solutions explained in a live workshop TODAY!
+                  USACO Bronze/Silver solutions explained in a live workshop
+                  TODAY!
```
Comment (ryanchou-dev) [inline src/components/TopNavigationBar/TopNavigationBar.tsx:189]: this alr passed right?

**Review by ryanchou-dev — APPROVED** [review]

File: src/components/TopNavigationBar/TopNavigationBar.tsx (hunk 1 above)
Comment (SansPapyrus683) [reply src/components/TopNavigationBar/TopNavigationBar.tsx:189]: wha

Comment (SansPapyrus683) [reply src/components/TopNavigationBar/TopNavigationBar.tsx:189]: how did this get in there

Comment (SansPapyrus683) [reply src/components/TopNavigationBar/TopNavigationBar.tsx:189]: alr i pulled from master, this should fix it

**Review by TruaShamu — APPROVED** [review]

**Pull request merged**